examples/command.rs

```diff
@@ -0,0 +1,642 @@
+#![allow(clippy::type_complexity)]
+
+use std::ops::DerefMut;
+
+use command::graph::CommandGraphBuilder;
+use command::handler::CommandResultEvent;
+use command::parsers::entity_selector::{EntitySelector, EntitySelectors};
+use command::parsers::{CommandArg, GreedyString, QuotableString};
+use command::scopes::CommandScopes;
+use command::{parsers, Command, CommandApp, CommandScopeRegistry, ModifierValue};
+use command_macros::Command;
+use parsers::{Vec2 as Vec2Parser, Vec3 as Vec3Parser};
+use rand::prelude::IteratorRandom;
+use valence::entity::living::LivingEntity;
+use valence::prelude::*;
+use valence::*;
+use valence_server::op_level::OpLevel;
+
+const SPAWN_Y: i32 = 64;
+
+#[derive(Command, Debug, Clone)]
+#[paths("teleport", "tp")]
+#[scopes("valence.command.teleport")]
+enum Teleport {
+    #[paths = "{location}"]
+    ExecutorToLocation { location: Vec3Parser },
+    #[paths = "{target}"]
+    ExecutorToTarget { target: EntitySelector },
+    #[paths = "{from} {to}"]
+    TargetToTarget {
+        from: EntitySelector,
+        to: EntitySelector,
+    },
+    #[paths = "{target} {location}"]
+    TargetToLocation {
+        target: EntitySelector,
+        location: Vec3Parser,
+    },
+}
+
+#[derive(Command, Debug, Clone)]
+#[paths("gamemode", "gm")]
+#[scopes("valence.command.gamemode")]
+enum Gamemode {
```

Comment:
Not sure this is the best command to use as an example because people like me will be wondering why we're not just using the normal `GameMode` parser.

Reply:
I like how the current version shows multiple paths or redirections. we could add a comment if we must.